### PR TITLE
Log format

### PR DESCRIPTION
**Pull request by alimez**

### CODE REVIEW

**Comment by alimez** [comment]
This PR lays the bed for stackdriver's structured logging in Turbinia. For now it only adds where the logs is generating from (server/worker) and we can add more custom fields to it as we go.

Sample query for server/worker:
```
jsonPayload.others="server"
```

File: turbinia/lib/google_cloud.py [hunk 1]
```diff
@@ -24,23 +24,53 @@
...
+
+def setup_stackdriver_handler(project_id, environment):
   """Set up Google Cloud Stackdriver Logging
 
   The Google Cloud Logging library will attach itself as a
   handler to the default Python logging module.
 
   Attributes:
     project_id: The name of the Google Cloud project.
+    environment: Where the log is running.
```
Comment (hacktobeer) [inline turbinia/lib/google_cloud.py:41]: maybe make this a little more explciti?  "Logging origin - 'server' or 'psqworker'"

**Review by hacktobeer — APPROVED** [review]
LGTM, small nit.

**Review by aarontp — APPROVED** [review]
LGTM (just the small nit mentioned in hacktober's review)

File: turbinia/lib/google_cloud.py (hunk 1 above)
Comment (alimez) [reply turbinia/lib/google_cloud.py]: fixed.

**Comment by alimez** [comment]
@aarontp PTAL

**Comment by aarontp** [comment]
LGTM

**Comment by alimez** [comment]
fixes #588

**Pull request merged**